testsuite/lvm1.cc

```diff
@@ -157,3 +157,55 @@ BOOST_AUTO_TEST_CASE(chunk_size_too_small)
     BOOST_CHECK_EQUAL(messages[0], "Chunk size is too small for thin pool logical volume "
 		      "thin-pool in volume group test.");
 }
+
+
+BOOST_AUTO_TEST_CASE(change_extent_size)
+{
+    Environment environment(true, ProbeMode::NONE, TargetMode::DIRECT);
+
+    Storage storage(environment);
+
+    Devicegraph* staging = storage.get_staging();
+
+    Disk* sda = Disk::create(staging, "/dev/sda", 1 * TiB);
+
+    LvmVg* lvm_vg = LvmVg::create(staging, "test");
+    lvm_vg->add_lvm_pv(sda);
+
+    LvmLv* normal1 = lvm_vg->create_lvm_lv("normal1", LvType::NORMAL, 1 * GiB);
+    LvmLv* normal2 = lvm_vg->create_lvm_lv("normal2", LvType::NORMAL, 6 * MiB);
+
+    BOOST_CHECK_EQUAL(lvm_vg->get_size(), 1 * TiB - 4 * MiB);
```

Comment:
Some explanation or a `define` for those 4 MiB would be nice; I can only assume that this is some metadata size that LVM needs for its own management; but that could be clearer.